immuno_tabular/code/make_clean_data.R

```diff
@@ -10,6 +10,8 @@ load(here("data_clean", "params.Rdata"))
 
 # Read in original data
 dat.mock <- read.csv(here("..", "data_clean", data_name))
+dat.mock$wt[is.na(dat.mock$wt)] <- 0
```

Comment:
@cyu-hvtn 
The new data pre-processing script sets some weights to `NA`, which was breaking the tabular code. Replacing those missing values with `0` seems to allow the tables to compile. Could you please comment on whether you think this change would affect the validity of the tables?